components/ca-import.html

```diff
@@ -850,4 +848,12 @@
     document.registerElement('ca-import', CaImportElement);
 
 });
+
+// private functions
+function camelCase(inputString) {
+    inputString.replace(/^([A-Z])|\s(\w)/g, (match, p1, p2, offset) => {
```

Comment:
So this would be:
return inputString.replace(/^([A-Z])|\s(\w)/g, (match, p1, p2, offset) => {
    if (p2) return p2.toUpperCase();
    return p1.toLowerCase();
});